cmd/crank/xpkg/batch.go

```diff
@@ -0,0 +1,574 @@
+/*
```

Comment:
This file is mostly all copied from its implementation in the `up` tool. The only differences here are:

* remove upbound context/flags
* use logger instead of pterm to be consistent with rest of crossplane CLI